bedrock/mozorg/templates/mozorg/rise25/landing.html

```diff
@@ -0,0 +1,243 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "base-protocol.html" %}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% block page_title %}Rise25 Awards: Call for Nominations{% endblock %}
+{% block page_desc %}We’re seeking 25 visionaries using AI to drive social impact, offering them global recognition at our Dublin awards event.{% endblock %}
+{% block page_image %}{{ static('img/mozorg/rise25/page-image.jpg') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-split') }}
+  {{ css_bundle('rise25') }}
+{% endblock %}
+
+{% block body_class %}rise25{% endblock %}
+
+{% set utm_params = '?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=rise25&utm_content=nominations-page' %}
+
+{% block content %}
+
+  {% call split(
+    block_class='mzp-l-split-center-on-sm-md r25-hero',
+    media_class='mzp-l-split-v-center',
+    media_include='mozorg/rise25/includes/hero-video.html',
+    media_after=True,
+  ) %}
+    <h1 class="r25-c-wordmark">Rise25</h1>
+    <h2 class="r25-fancy-title">AI for the people</h2>
+    <p class="r25-hero-intro">We’re seeking 25 visionaries using AI to drive social
+      impact, offering them global recognition at our Dublin awards event. By sharing
+      their stories widely, we’ll amplify their work and connect them with a community
+      dedicated to supporting responsible AI initiatives that benefit humanity. Know
+      an innovator making a difference with AI?</p>
+    <p class="r25-hero-cta">
```

Comment:
**Suggestion:** In smaller screens, this section is too tight with the video and the CTA. Can we vertically align these two elements instead of horizontally?

![Screenshot 2024-02-27 at 10 45 34 AM](https://github.com/mozilla/bedrock/assets/42309026/61fa12d2-87f8-47b8-9669-581bb549e663)